vineyard/data/stock_location.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="stock_location_33" model="stock.location">
```

Comment:
Could you check with Pierre which location should be in data and which one in demo?